app/models/rss_log.rb

```diff
@@ -387,6 +387,11 @@ def detail
     ""
   end
 
+  # Has target been destroyed (orphaning this log)?
```

Comment:
Is `orphan?` intended to be part of the class's public interface?  (I see that it's called by the tests.)
- If no, consider either:
  - Moving it to the private section, and in the tests accessing it with `send`
  - At least moving it the next section
- If yes, document it with other methods (after line 136).

Reply:
Yes, I did intend to make it public even though we have to potential uses for it yet.  If I had to articulate my feelings on the subject... I guess I prefer to make public, potentially useful methods over private, highly-specialized methods.  Yes, it's the same exact method!  But what was target_simply_deleted? doing?  It was in effect checking to see if the log is an orphan.  Well, hell, that's a really useful piece of information lots of people might want access to.  But no one would think to use "target_simply_deleted?"  That makes sense -- and it makes great sense, I admit! -- in the context of RssLog#detail, but outside of that context I would personally wonder about the choice of wording.  I would assume it was doing something highly specific and customized for its one caller.  It's not, but I would have to look up the code to figure that out.  Then, imagine code elsewhere that calls rss_log.target_simply_deleted? in a different context -- say, a script which culls stale orphan logs.  That method name would make no sense in that context.  But if we rename it orphan?, yes it requires a bit more thought inside RssLog#detail, but you can figure out what it does easily enough, and you would quickly understand why you were using it there.  *And* it would work well in other contexts like the orphan log culler.

Oh I got it!  Another way to look at it that might make more sense is that you were naming the method according to what the *caller* wants it to do, not according to what the *method* is doing.  Does that make sense?  The caller wants a method that tells if if the target was simply deleted.  That happens to work out to checking if the log is an orphan.  The method doesn't have the context, all it knows that it's doing is checking if its an orphan.  "Simply deleted" implies that there are alternatives such as complicatedly deleted, or "deleted and something else", or ... who knows?  It's really kind of vague on what "simply" means!  "Orphan?" on the other hand is easily understood out of context.  It describes perfectly what the *method* is doing, not what the first caller happened to need it for.